tests/metrics/network/dockerfile_image/README.md

```diff
@@ -0,0 +1,18 @@
+# Networking image for Intel® Clear Containers performance metrics
+
+If you want to build the network image with the general tooling
+and system dependencies that we are using in our networking tests,
+just do the following:
```

Comment:
Please change the paragraph to:
In order to build the network image with the general tooling
 and system dependencies which are used in our networking tests, follow these steps: